usecases/traverser/explorer.go

```diff
@@ -61,14 +62,23 @@ type explorerMetrics interface {
 	AddUsageDimensions(className, queryType, operation string, dims int)
 }
 
+type ModulesMultiVectorProvider interface {
+	MultiVectorFromSearchParam(ctx context.Context, className, targetVector, tenant, param string, params interface{},
+		findVectorFn modulecapabilities.FindVectorFn[[][]float32]) ([][]float32, error)
+	MultiCrossClassVectorFromSearchParam(ctx context.Context, param string,
+		params interface{}, findVectorFn modulecapabilities.FindVectorFn[[][]float32]) ([][]float32, string, error)
+	MultiVectorFromInput(ctx context.Context, className, input, targetVector string) ([][]float32, error)
+}
+
 type ModulesProvider interface {
+	ModulesMultiVectorProvider
```

Comment:
Q: I don't see the point of embedding here with `ModulesMultiVectorProvider`, Because
1. Anyway `ModulesProvider` is going to have those methods (plus same `modules.Provider` is going to provide those methods with implementations).
2. Plus it's not used anywhere else.

Can we remove the embeddings and add methods directly to `ModuleProvider`?  Am I missing anything?

Reply:
Yes, I'll do that in my next commit

Reply:
done, check [here](https://github.com/weaviate/weaviate/pull/6446/commits/fab9139c54081761e582e89548fde6da0e5432cb)